### PR TITLE
Updates to the Flex 4 applications

### PR DESCRIPTION
**Pull request by GhaziTriki**
- Deleted `style` suffix for styles.
- Upgraded to AIR 23.0.
- Deleted CustomRuntimeDPIProvider.
- Created multiple CSS file for the different DPI's.
- Implement MainView styles.

### CODE REVIEW

File: clients/flash/air-client/src/Main-app.xml [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
-
-		Specifies parameters for identifying, installing, and launching AIR applications.
-
-		xmlns - The Adobe AIR namespace: http://ns.adobe.com/air/application/3.1
-				The last segment of the namespace specifies the version
-				of the AIR runtime required for this application to run.
-
-		minimumPatchLevel - The minimum patch level of the AIR runtime required to run
-				the application. Optional.
+<application xmlns="http://ns.adobe.com/air/application/23.0">
```
Comment (capilkey) [inline clients/flash/air-client/src/Main-app.xml]: The only change in this file appears to be this version change, but there are a bunch of other diffs that don't appear to change anything.

File: clients/flash/air-client/src/org/bigbluebutton/air/main/views/PagesNavigatorViewMediator.as [hunk 2]
```diff
@@ -1,93 +1,93 @@
-package org.bigbluebutton.air.main.views {
```
Comment (capilkey) [inline clients/flash/air-client/src/org/bigbluebutton/air/main/views/PagesNavigatorViewMediator.as]: This whole file is changed, but I can't find any actual differences.

File: clients/flash/common-library/src/main/actionscript/org/bigbluebutton/lib/main/commands/ConnectCommand.as [hunk 3]
```diff
@@ -1,5 +1,5 @@
 package org.bigbluebutton.lib.main.commands {
-
```
Comment (capilkey) [inline clients/flash/common-library/src/main/actionscript/org/bigbluebutton/lib/main/commands/ConnectCommand.as]: The FlexFormatter definition includes tabs on the empty lines, but the edit to this file removes them all.

File: clients/flash/common-library/src/main/actionscript/org/bigbluebutton/lib/user/services/UsersMessageReceiver.as [hunk 4]
```diff
@@ -1,28 +1,29 @@
 package org.bigbluebutton.lib.user.services {
 	import mx.utils.ObjectUtil;
-
```
Comment (capilkey) [inline clients/flash/common-library/src/main/actionscript/org/bigbluebutton/lib/user/services/UsersMessageReceiver.as]: This file was edited to remove the tabs on empty lines, but the FlexFormatter definition includes them.

File: clients/flash/air-client/src/org/bigbluebutton/air/main/views/PagesNavigatorViewMediator.as (hunk 2 above)
Comment (GhaziTriki) [inline clients/flash/air-client/src/org/bigbluebutton/air/main/views/PagesNavigatorViewMediator.as]: The file EOL was linux, and it was converted to windows when I formatted it.

**Comment by GhaziTriki** [comment]
Please don't merge yet. There is some display issues that needs to be fixed first.

**Comment by GhaziTriki** [comment]
Display issue fixed.

**Pull request merged**